fe/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -270,8 +281,20 @@ protected void setOptional(CreateRoutineLoadStmt stmt) throws UserException {
         if (stmt.getMaxBatchSize() != -1) {
             this.maxBatchSizeBytes = stmt.getMaxBatchSize();
         }
+        if (stmt.getFormat().equals("json")) {
```

Comment:
stmt.getFormat() may null in csv format

Reply:
No， the class field `format` is initial "" in CreateRoutineLoadStmt.java, then it is safely

Reply:
![image](https://user-images.githubusercontent.com/9495145/79560947-195deb00-80db-11ea-8ce4-98841e15b72f.png)

Reply:
I think format to null is not handled in CreateRoutineLoadStmt